src/sheet_metal_conus.inx

```diff
@@ -26,7 +27,7 @@ If Cut diameter=0 then cone will not be truncated.</param>
   <effect>
     <object-type>all</object-type>
     <effects-menu>
-       <submenu _name="Render"/>
+       <submenu _name="Folded Forms"/>
```

Comment:
I think this is a good place for this and several other objects made from flat sheets which can be folded up into complex forms. I hope you agree ?